CHANGELOG

```diff
@@ -1,3 +1,6 @@
+To be released
+    - Add background-color mixin
+    - Deprecate `base-bg-color`, `bg-color-neutral-30`, `bg-color-error`, and `bg-color-light-error` mixins
```

Comment:
Technically, those mixins weren't so much deprecated as were outright removed. There is an important distinction: a deprecated function (or mixin, etc.) is simply altered in a way that demonstrates its disapproval for product use (for example, outputs a deprecation warning) but still works.